### DISCUSSION TITLE
The cutting plane algorithm is not effective in solving the scheduling problem of single machine(linear ordering) for highs，SCIP，CBC

### DISCUSSION THREAD

**Discussion by tusiqi1** [ERGO-Code/HiGHS#1421] [General]
The corresponding example in MIPLIB is [neos-3046615-murg](https://miplib.zib.de/instance_details_neos-3046615-murg.html) which is labeled as simple.

cplex、gurobi can get a better lower bound fast. 

What improvements might make the lower bound rise faster?

**Comment by jajhall** [comment]
Techniques developed in Gurobi and CPLEX with man years of effort and experience that we don't have! 😁

Pay us a lot of money, and we might improve on this class of MIP! 🙂

**Reply by tusiqi1** [reply]
Thanks for professor's reply. This phenomenon was discovered during my research. 

I believe we will have cooperation in the future! 

But I'm just a researcher and I don't have the right to decide where the money is spent right now, sorry.
